README.md

```diff
@@ -1,28 +1,11 @@
 # CodeStyle [![Build Status](https://travis-ci.com/eigen-space/codestyle.svg?branch=master)](https://travis-ci.com/eigen-space/codestyle)
 
-Main language: `TypeScript`
-
 This project contains:
 1. Code writing guide. Covers aesthetic issues, convention naming and coding standards.
-2. Rules for `tslint` and its configuration.
-3. tsconfig configuration - `codestyle.tsconfig.json`.
+2. eslint configuration.
+3. tsconfig configuration - `base.tsconfig.json`.
 4. Configuration for development environments.
 
-# Structure
```

Comment:
We have lost section that describes project structure.

Reply:
I deleted it. Do we really need her?